### PR TITLE
P16 recovered ingestion csvs

### PR DESCRIPTION
**Pull request by cdobs**
Creates recovered ingestion csvs for:
-CP01CNSM R00014
-CP02PMCO R00015
-CP02PMUO R00017
-CP03ISSM R00013
-CP04OSPM R00014
-CP04OSSM R00013

### CODE REVIEW

File: CP02PMCO/CP02PMCO_R00015_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00015/instruments/CWFP_sn*/*/E*.DA?,CP02PMCO-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00015/instruments/CWFP_sn*/*/A*.DA?,CP02PMCO-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00015/instruments/CWFP_sn*/*/C*.DA?,CP02PMCO-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00015/instruments/CWFP_sn*/*/C*.DA?,CP02PMCO-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00015/instruments/CWFP_sn*/*/E*.DA?,CP02PMCO-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00015/instruments/CWFP_sn*/*/E*.DA?,CP02PMCO-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (johnmlund) [inline CP02PMCO/CP02PMCO_R00015_ingest.csv:11]: Looks good

File: CP02PMUO/CP02PMUO_R00017_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00017/instruments/CWFP_sn*/*/E*.DA?,CP02PMUO-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00017/instruments/CWFP_sn*/*/A*.DA?,CP02PMUO-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00017/instruments/CWFP_sn*/*/C*.DA?,CP02PMUO-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00017/instruments/CWFP_sn*/*/C*.DA?,CP02PMUO-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00017/instruments/CWFP_sn*/*/E*.DA?,CP02PMUO-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00017/instruments/CWFP_sn*/*/E*.DA?,CP02PMUO-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (johnmlund) [inline CP02PMUO/CP02PMUO_R00017_ingest.csv:11]: looks good. confirmed no mopak data.

File: CP04OSPM/CP04OSPM_R00014_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00014/instruments/CWFP_sn*/*/E*.DA?,CP04OSPM-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00014/instruments/CWFP_sn*/*/A*.DA?,CP04OSPM-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00014/instruments/CWFP_sn*/*/C*.DA?,CP04OSPM-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00014/instruments/CWFP_sn*/*/C*.DA?,CP04OSPM-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00014/instruments/CWFP_sn*/*/E*.DA?,CP04OSPM-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00014/instruments/CWFP_sn*/*/E*.DA?,CP04OSPM-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (johnmlund) [inline CP04OSPM/CP04OSPM_R00014_ingest.csv:11]: looks good

**Review by johnmlund — APPROVED** [review]
Profiler Moorings reviewed and look Okay.

File: CP01CNSM/CP01CNSM_R00014_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00014/cg_data/dcl12/hyd2/*.hyd2.log,CP01CNSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00014/cg_data/dcl12/pco2a/*pco2a.log,CP01CNSM-SBD12-04-PCO2AA000,recovered_host,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00014/cg_data/dcl12/wavss/*.wavss.log,CP01CNSM-SBD12-05-WAVSSA000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00014/cg_data/dcl12/metbk2/*metbk2.log,CP01CNSM-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00014/cg_data/dcl12/fdchp/*fdchp.log,CP01CNSM-SBD12-08-FDCHPA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00014/instruments/dcl12/FDCHP_sn*/D20*/fdchp*.dat,CP01CNSM-SBD12-08-FDCHPA000,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline CP01CNSM/CP01CNSM_R00014_ingest.csv:46]: looks good - data available & aligns w/ data recovery checklist

File: CP02PMCO/CP02PMCO_R00015_ingest.csv (hunk 1 above)
Comment (rtravis-whoi) [inline CP02PMCO/CP02PMCO_R00015_ingest.csv:11]: looks good - data available & aligns w/ data recovery checklist

File: CP02PMUO/CP02PMUO_R00017_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
```
Comment (rtravis-whoi) [inline CP02PMUO/CP02PMUO_R00017_ingest.csv:11]: looks good - data available & aligns w/ data recovery checklist; no mopak data - line commented out

File: CP03ISSM/CP03ISSM_R00013_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00013/cg_data/dcl11/mopak/*.mopak.log,CP03ISSM-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00013/cg_data/dcl11/hyd1/*.hyd1.log,CP03ISSM-SBD11-02-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00013/cg_data/dcl11/metbk/*.metbk.log,CP03ISSM-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00013/cg_data/dcl12/syslog/*.syslog.log,CP03ISSM-SBD12-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00013/cg_data/dcl12/hyd2/*.hyd2.log,CP03ISSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00013/cg_data/dcl12/pco2a/*pco2a.log,CP03ISSM-SBD12-04-PCO2AA000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP03ISSM/CP03ISSM_R00013_ingest.csv:42]: looks good - data available & aligns w/ data recovery checklist

File: CP04OSPM/CP04OSPM_R00014_ingest.csv (hunk 3 above)
Comment (rtravis-whoi) [inline CP04OSPM/CP04OSPM_R00014_ingest.csv:11]: looks good - data available & aligns w/ data recovery checklist

File: CP04OSSM/CP04OSSM_R00013_ingest.csv [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/instruments/dcl11/FDCHP_sn*/D20*/fdchp*.dat,CP04OSSM-SBD11-08-FDCHPB000,recovered_inst,Available,test deployment; R3-50 anemometer
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/hyd2/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/pco2a/*pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/fdchp1/*fdchp1.log,CP04OSSM-SBD12-08-FDCHPA000,recovered_host,Available,test deployment
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/instruments/dcl12/FDCHP_sn*/D20*/fdchp*.dat,CP04OSSM-SBD12-08-FDCHPA000,recovered_inst,Available,test deployment
```
Comment (rtravis-whoi) [inline CP04OSSM/CP04OSSM_R00013_ingest.csv:46]: looks good - data available & aligns w/ data recovery checklist; no pco2w instrument data as noted on data recovery checklist & line commented out.

**Review by rtravis-whoi — COMMENTED** [review]
PMs & CNSM, ISSM look good; have question re test FDCHPs on OSSM - that is pending more discussion during Thurs data meeting.

File: CP01CNSM/CP01CNSM_R00014_ingest.csv (hunk 4 above)
Comment (snwhite) [reply CP01CNSM/CP01CNSM_R00014_ingest.csv:46]: Looks good.

File: CP03ISSM/CP03ISSM_R00013_ingest.csv (hunk 6 above)
Comment (snwhite) [reply CP03ISSM/CP03ISSM_R00013_ingest.csv:42]: Looks good.

File: CP04OSSM/CP04OSSM_R00013_ingest.csv (hunk 7 above)
Comment (snwhite) [reply CP04OSSM/CP04OSSM_R00013_ingest.csv:46]: Looks good, except that FDCHPs should be commented out (test deployment)

**Review by snwhite — COMMENTED** [review]
3 Surface Moorings approved.  Test FDCHPs should be commented out.

File: CP04OSSM/CP04OSSM_R00013_ingest.csv [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/mopak/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/hyd1/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/metbk/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/hyd2/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/pco2a/*pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP04OSSM/CP04OSSM_R00013_ingest.csv:42]: update looks good; test FDCHP lines removed per data mtg discussion.

Comment (snwhite) [inline CP04OSSM/CP04OSSM_R00013_ingest.csv:42]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**